### PR TITLE
updated all gems to install rails 6

### PR DESCRIPTION
**Pull request by andreorvalho**
This is a start of rails 6 upgrade

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-gem 'rspec-collection_matchers', '~> 1.1'
-gem 'rdoc', '~> 4.2'
+gem 'rake', '~> 13'
+gem 'rspec-collection_matchers', '~> 1.2'
+gem 'rdoc', '~> 6.2.1'
 gem 'simplecov', :platforms => :mri_19
 
 group :multipart_support do
-  gem 'multipart-post', '~> 2'
+  gem 'multipart-post'
```
Comment (corthmann) [inline Gemfile]: we need this fixed as I don't think wrest works with 1.x of this gem. I'd suggest:
```suggestion
  gem 'multipart-post',  '>= 2.0', '< 3.0'
```

File: lib/wrest/caching/redis.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 begin
-  gem 'redis', '~> 3'
+  gem 'redis', '~> 4'
```
Comment (corthmann) [inline lib/wrest/caching/redis.rb:2]: the comment below is now untrue

**Review by corthmann — CHANGES_REQUESTED** [review]
Nice to get dependencies updated here :heart:

**Review by corthmann — APPROVED** [review]

**Review by ksza — APPROVED** [review]
👍

**Comment by andersklenke** [comment]
We're upgrading the Rails version to 6.1 in wine-admin. Can we have this PR merged?

**Pull request merged**